browser/js/calendar/calendar.html

```diff
@@ -1,18 +1,32 @@
-<div class="container" ng-controller="CalendarCtrl">
-  <calendar-md flex layout layout-fill
-    calendar-direction="direction"
-    on-prev-month="prevMonth"
-    on-next-month="nextMonth"
-    on-day-click="dayClick"
-    title-format="'MMMM y'"
-    ng-model='selectedDate'
-    week-starts-on="firstDayOfWeek"
-    tooltips="tooltips"
-    day-format="dayFormat"
-    day-label-format="'EEE'"
-    day-label-tooltip-format="'EEEE'"
-    day-tooltip-format="'fullDate'"
-    day-content="setDayContent"
-    disable-future-selection="false">
-  </calendar-md>
+<div ng-controller="CalendarCtrl">
+    <div class="row">
+        <div class="col s12">
+            <h1>Harvest Calendar</h1>
+        </div>
+    </div>
+
+    <div class="row bottom_spacing-medium">
+        <div class="col s12">
+            <hr>
+        </div>
+    </div>
+
+    <div class="col s12">
```

Comment:
confused as to why git is marking a lot of these things as new (like lines 15-31) - were they copy and pasted or otherwise shifted around in a weird way? it makes it really hard to review the PR because I don't know what was actually changed.